lib/strategy/unordered.ts

```diff
@@ -14,11 +19,14 @@ export class UnorderedStrategy {
     private inFlight: number = 0;
 
     private fetchNotifier: Notifier<FetchEvent, { index: number }>;
-    private memberNotifier: Notifier<MemberEvents, unknown>;
+    private memberNotifier: Notifier<
+        MemberEvents,
+        { index: number, emitted: ReadonlySet<string> }
+    >;
 
-    private modulator: Modulator<Node>;
+    private modulator: Modulator<Node, SerializedMember>;
 
-    private cacheList: Node[] = [];
+    private toPoll: Node[] = [];
```

Comment:
is `toPoll` not the same as `modulator.getMutable()` ?
If not, this needs a comment imo, you add them at the same time, so the differentiating flow is that when the client polls, the fetched pages are added again to toPoll if not immutable. If a page is not immutable it would be added to the modulator as well, and if a page _is_ immutable it would be added to the modulator as immutable which removes it from mutable, thus same behavior

Reply:
Indeed, this is redundant in the `unordered` strategy and it has been removed in https://github.com/rdf-connect/ldes-client/pull/94/commits/048030e82a501e0f8cd7c2029929b3ed4fad9b9f. We still need it in the `ordered` strategy though, because there we need to guarantee the order with a Heap.